pkg/packages/install.go

```diff
@@ -173,36 +175,31 @@ type InstallOrUpdateBinaryOptions struct {
 }
 
 // ShouldInstallBinary checks if the given binary should be installed
-func ShouldInstallBinary(name string) (fileName string, download bool, err error) {
-	fileName = BinaryWithExtension(name)
-	download = false
-	pgmPath, err := exec.LookPath(fileName)
```

Comment:
one thing we did not consider is when this runs in cluster. There we know that the binaries are in the right version and we probably want to avoid downloading the binaries each time a pipeline step needs to call an external binary. 

Ideally, these binaries would be installed into _JX_HOME_ in the images as well ,but that's not the case. I am wondering whether we don't have to keep the `exec.LookPath` call but wrap it in a `cluster.IsInCluster()` check. So only if we are in cluster we do a `exec.LookPath` and returns _false_ if the binary can be found on the global path.

When not in cluster, aka on the users machine, I still believe we want to make sure all binaries we need are installed in the version we require into _JX_HOME_.

Reply:
Hey @hferentschik thanks! I won't be able to complete this PR as off now. Can we please close it, I'll work on it (if open) after sometime? 
Apologies for the delay.